File: app/src/main/java/com/zuk0/gaijinsmash/riderz/data/local/entity/trip_response/Schedule.java
package com.zuk0.gaijinsmash.riderz.data.local.entity.trip_response;

import org.simpleframework.xml.Element;

public class Schedule {

    @Element
    String date;

    @Element
    String time;

    @Element
    private Request request;

    public Request getRequest() { return request; }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public String getTime() {
        return time;
    }

    public void setTime(String time) {
        this.time = time;
    }

    public void setRequest(Request request) {
        this.request = request;
    }
}

File: app/src/main/java/com/zuk0/gaijinsmash/riderz/data/local/entity/trip_response/Schedule.kt
package com.zuk0.gaijinsmash.riderz.data.local.entity.trip_response

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Schedule {
    @SerializedName("date")
    @Expose
    var date: String? = null
    @SerializedName("time")
    @Expose
    var time: String? = null
    @SerializedName("before")
    @Expose
    var before: String? = null
    @SerializedName("after")
    @Expose
    var after: String? = null
    @SerializedName("request")
    @Expose
    var request: Request? = null

}